mail_tracking_mailgun/README.rst

```diff
@@ -29,10 +29,13 @@ You must configure Mailgun webhooks in order to receive mail events:
 Replace '<your_domain>' with your Odoo install domain name
 and '<your_database>' with your database name.
 
-In order to validate Mailgun webhooks you have to configure the following system parameters:
+In order to validate Mailgun webhooks you have to configure the following system
+parameters:
 
 `mailgun.apikey`: You can find Mailgun api_key in your validated sending domain.
 `mailgun.api_url`: It should be fine as it is, but it could change in the future.
+`mailgun.validation_key`: If you want to be able to check mail address validity
```

Comment:
I think this should be a [bullet list](http://docutils.sourceforge.net/docs/user/rst/quickref.html#bullet-lists)